### PR TITLE
Fix unittest issue

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

File: tests/run_tests.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#===============================================================================
+# Copyright 2020-2021 Intel Corporation
```
Comment (PetrovKP) [inline tests/run_tests.py:2]:
```suggestion
# Copyright 2021 Intel Corporation
```

File: tests/run_tests.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#===============================================================================
+
+import sys
+import unittest
+
+if __name__ == '__main__':
+    test_suite = unittest.defaultTestLoader.discover('tests', 'test*.py')
+    test_runner = unittest.TextTestRunner()
+    result = test_runner.run(test_suite)
+    sys.exit(not result.wasSuccessful())
```
Comment (PetrovKP) [inline tests/run_tests.py:24]: warnings will red ci?

Comment (OnlyDeniko) [reply tests/run_tests.py:24]: No

**Review by PetrovKP — APPROVED** [review]
Update here, pls https://github.com/oneapi-src/oneDAL/blob/master/.ci/pipeline/ci.yml#L173

**Pull request merged**